api/.project/devstart.rb

```diff
@@ -0,0 +1,51 @@
+require_relative "common/common"
+
+def startdb()
+  c = Common.new
```

Comment:
I guess this stands for, "common things you are likely to want to do in most scripts." I could call it `common` but that doesn't actually seem any more useful than `c`. Is it?